### PR TITLE
feat(playground): add new router view

### PR DESCRIPTION
**Pull request by alili**

### CODE REVIEW

File: src/views/dashboard/playground/docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+​
+
+```sql
+-- Type your SQL here
+​
+```
+
+​
+Full more advanced features like scripting, various protocol support,
+[Download](https://greptime.com/downloads/) GreptimeDB on your machine and
```
Comment (sunng87) [inline src/views/dashboard/playground/docs/getting-started.md:127]:
```suggestion
[Download](https://greptime.com/download/) GreptimeDB on your machine and
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+Enough with the basics, try to write your own query and do some exploration.
+​
+
+```sql
+-- Type your SQL here
+​
+```
+
+​
+Full more advanced features like scripting, various protocol support,
```
Comment (sunng87) [inline src/views/dashboard/playground/docs/getting-started.md:126]:
```suggestion
For more advanced features like scripting and protocol support,
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    cpu_util DOUBLE,
+    memory_util DOUBLE,
+    disk_util DOUBLE,
+    ts TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY(host, idc),
+    TIME INDEX(ts)
+);
+```
+
+Once you see the green marker, run `DESC TABLE` to check detail of the table.
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:38]:
```suggestion
Once the green marker appears, run `DESC TABLE` to view the details of the table.
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+In GreptimeDB, there are three kinds of column in term of semantic type:
+
+- `PRIMARY KEY`: Key columns used for sorting and partition
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:42]:
```suggestion
- `PRIMARY KEY`: key columns that are used for sorting and partitioning
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+- `FIELD`: Value columns that hold values
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:43]:
```suggestion
- `FIELD`: columns which store values
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+- `TIME INDEX`: The time column
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:44]:
```suggestion
- `TIME INDEX`: columns of data type TIME
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+
+```sql
+DESC TABLE system_metrics;
+```
+
+## Add Some Data
+
+Using the `INSERT` statement is an easy way to add data to your table. Through
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:52]:
```suggestion
Using the `INSERT` statement to easily add data to the table. 
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+the statement below, we have inserted three rows into the `system_metrics`
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:53]:
```suggestion
Below example inserts three rows into the `system_metrics` 
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+``` sql
+INSERT INTO system_metrics
+VALUES
+    ("host1", "idc_a", 11.8, 10.3, 10.3, 1667446791450),
+    ("host2", "idc_a", 80.1, 70.3, 90.0, 1667446792460),
+    ("host1", "idc_b", 50.0, 66.7, 40.6, 1667446793470);
+```
+
+
+Modify the value and timestamp, we can ingest more entries.
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:66]:
```suggestion
Ingest more entries by modifying the values and timestamps.
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+    ("host2", "idc_a", 60.8, 74.9, 96.2, 1667446795490),
+    ("host1", "idc_b", 30.0, 65.7, 40.2, 1667446796500);
+```
+
+
+
+## Query Data with SQL
+
+
+GreptimeDB supports full SQL for you to query data from a database.
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:82]:
```suggestion
GreptimeDB fully supports SQL.
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 11]
```diff
@@ -0,0 +1,133 @@
...
+
+
+``` sql
+SELECT * FROM system_metrics ORDER BY ts DESC;
+```
+
+
+Here are some query examples for the `system_metrics` so you can get familiar
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:90]:
```suggestion
Here are some query examples for the `system_metrics` for users to get familiar
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 12]
```diff
@@ -0,0 +1,133 @@
...
+with using SQL alongside GreptimeDB functions.
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:91]:
```suggestion
with using SQL alongside GreptimeDB's functions.
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 13]
```diff
@@ -0,0 +1,133 @@
...
+
+Use `avg()` function returns the average value of a certain field:
+
+
+``` sql
+SELECT avg(cpu_util) FROM system_metrics;
+```
+
+
+You can use the `GROUP BY` clause to group rows that have the same values into
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:109]:
```suggestion
Use the `GROUP BY` clause to group rows that have the same values into
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 14]
```diff
@@ -0,0 +1,133 @@
...
+```sql
+SELECT idc, avg(memory_util) FROM system_metrics GROUP BY idc;
+```
+
+
+
+## Explore by Yourself
+
+
+Enough with the basics, try to write your own query and do some exploration.
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:122]:
```suggestion
Please start exploring by writing some queries!
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 15]
```diff
@@ -0,0 +1,133 @@
...
+
+
+```sql
+-- Type your SQL here
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:126]:
```suggestion
-- Type SQL statements here
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 16]
```diff
@@ -0,0 +1,133 @@
...
+
+```
+
+
+For more advanced features like scripting and protocol support,
+[Download](https://greptime.com/download/) GreptimeDB on your machine and
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:132]:
```suggestion
[Download](https://greptime.com/download/) and run GreptimeDB locally by
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 17]
```diff
@@ -0,0 +1,133 @@
...
+follow our [docs](https://docs.greptime.com).
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:133]:
```suggestion
following [docs](https://docs.greptime.com).
```

**Review by apdong2022 — CHANGES_REQUESTED** [review]

**Comment by ZonaHex** [comment]
Great work!
A couple of suggestions: 
- [ ] Delete default select from the editor. (Will do it in the main branch).
- [ ] Captcha seems to be slowing down the main page loading. Perhaps we could move it into the playground only.
- [ ] Use line(smooth) as default in Chart option. (Will do it in the main branch).

File: src/views/dashboard/playground/docs/getting-started.md [hunk 18]
```diff
@@ -0,0 +1,135 @@
...
+```
+
+Use `avg()` function returns the average value of a certain field:
+
+``` sql
+SELECT avg(cpu_util) FROM system_metrics;
+```
+
+Use the `GROUP BY` clause to group rows that have the same values into
+summary rows. The average memory usage grouped by `idc`:
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:107]:
```suggestion
summary rows. The code below groups average memory usage by `idc`:
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 19]
```diff
@@ -0,0 +1,135 @@
...
+
+There are more aggregate function available, change `avg` to any of these below
+and try out:
+
+- `max`
+- `min`
+- `sum`
+- `mean`
+
+## Explore by Yourself
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:122]:
```suggestion
## Try it Out
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 20]
```diff
@@ -0,0 +1,135 @@
...
+
+
+Please start exploring by writing some queries!
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:125]:
```suggestion
Please start exploring by writing some queries in the panel below!
```

File: src/views/dashboard/playground/docs/getting-started.md [hunk 21]
```diff
@@ -0,0 +1,135 @@
...
+
+
+```sql
+
+```
+
+
+For more advanced features like scripting and protocol support,
```
Comment (apdong2022) [inline src/views/dashboard/playground/docs/getting-started.md:133]:
```suggestion
For other advanced features like scripting and protocol supports,
```

**Review by apdong2022 — CHANGES_REQUESTED** [review]
fine tuned the wording of Greptime Play by referring to the tool

**Review by ZonaHex — COMMENTED** [review]
Great work!
A couple of suggestions: 
- [x] Delete default select from the editor. (Will do it in the main branch).
- [x] Captcha seems to be slowing down the main page loading. Perhaps we could move it into the playground only.
- [x] Use line(smooth) as default in Chart option. (Will do it in the main branch).
- [x] After Deleting the last result tab, it should not go back to the first tab.
- [ ] 'Vue warn' in the console after clicking 'run'.

**Review by sunng87 — APPROVED** [review]

**Pull request merged**